.github/workflows/nox.yml

```diff
@@ -0,0 +1,72 @@
+---
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2023 Maxwell G <maxwell@gtmx.me
+
+name: nox
+'on':
+  push:
+    branches:
+      - main
+      - stable-*
+  pull_request:
+    branches:
+      - main
+      - stable-*
+  # Run once per week (Monday at 04:00 UTC)
+  schedule:
+    - cron: '0 4 * * 1'
+  workflow_dispatch:
+
+jobs:
+  nox-lint:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: antsibull-docs
+    steps:
+      - name: Check out antsibull-docs
+        uses: actions/checkout@v3
+        with:
+          path: antsibull-docs
+      - name: Check out dependent project antsibull-core
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/antsibull-core
+          path: antsibull-core
```

Comment:
You can remove this if you'd like, as nox already handles installing antsibull-core from `main`.

Reply:
I think it's easier to leave it here so if we ever need to check out a specific branch (instead of `main`), it's a one-line change here, instead of adding this whole block first.

Reply:
I was also thinking that.

/resolve